### ISSUE TITLE
[Feature]: Avatar from address based on barcode

### ISSUE THREAD

**Issue by benbms2** [opensourcepos/opensourcepos#4141]
### Type of Feature

✨ New Feature

### OpensourcePOS Version

opensourcepos 3.3.9

### Description

Hi,
Just wondering if there is any way to have the avatars as a url that is auto filled with the barcode. We are looking at using the Open library covers api (https://openlibrary.org/dev/docs/api/covers) to pull iamges of book covers so all we would need to do is automatically fill the avatar field with an address (https://covers.openlibrary.org/b/isbn/9780385533225-S.jpg) incorporating the barcode number. 

### Verify you searched open requests in OpensourcePOS

- [X] I agree I have searched Open Requests

**Comment by andriux1990** [comment]
Yes, it’s possible. When an image is uploaded, what gets saved in the database, specifically in the ospos_items table under the pic_filename column, is only the image file name.

To make it save the full path, a few adjustments are needed:
Replace the current upload method with a traditional input field to store the full path.
In the code, remove the reference to the predefined directory where the image is being looked for, and instead use the full URL of the cover image.

This can be implemented, but it requires making some changes on yours code.
